jetbrains-core/src/software/aws/toolkits/jetbrains/services/ecs/exec/RunCommandToolCustomizer.kt

```diff
@@ -0,0 +1,23 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.ecs.exec
+
+import com.intellij.execution.configurations.GeneralCommandLine
+import com.intellij.openapi.actionSystem.CommonDataKeys
+import com.intellij.openapi.actionSystem.DataContext
+import com.intellij.tools.ToolsCustomizer
+import software.aws.toolkits.jetbrains.core.credentials.AwsConnectionManager
+import software.aws.toolkits.jetbrains.core.credentials.toEnvironmentVariables
+
+class RunCommandToolCustomizer : ToolsCustomizer() {
+    override fun customizeCommandLine(dataContext: DataContext, commandLine: GeneralCommandLine): GeneralCommandLine {
```

Comment:
This will affect any and all tools, which we should avoid

Reply:
Removed the customizer